utils/keydb_init_script

```diff
@@ -1,30 +1,30 @@
 #!/bin/sh
 #
-# Simple Redis init.d script conceived to work on Linux systems
+# Simple KeyDB init.d script conceived to work on Linux systems
 # as it does use of the /proc filesystem.
 
 ### BEGIN INIT INFO
 # Provides:     redis_6379
 # Default-Start:        2 3 4 5
 # Default-Stop:         0 1 6
-# Short-Description:    Redis data structure server
-# Description:          Redis data structure server. See https://redis.io
+# Short-Description:    KeyDB data structure server
+# Description:          KeyDB data structure server. See https://redis.io
 ### END INIT INFO
 
-REDISPORT=6379
+KEYDBPORT=6379
 EXEC=/usr/local/bin/keydb-server
 CLIEXEC=/usr/local/bin/keydb-cli
 
-PIDFILE=/var/run/redis_${REDISPORT}.pid
-CONF="/etc/redis/${REDISPORT}.conf"
+PIDFILE=/var/run/redis_${KEYDBPORT}.pid
+CONF="/etc/redis/${KEYDBPORT}.conf"
```

Comment:
Presumably these dirs should change? @JohnSully

Reply:
Yea good point.